pypff-stubs/pypff/file.pyi

```diff
@@ -0,0 +1,125 @@
+"""
+Type stubs for pypff_file, defined in [1]
+
+[1] https://github.com/Everlaw/libpff/blob/master/pypff/pypff_file.c
```

Comment:
Just to double check - these were created from our frozen version, rather than the upstream version, right?